ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/QueryPlanningTool.java

```diff
@@ -63,15 +62,15 @@ public <T> void run(Map<String, String> parameters, ActionListener<T> listener)
             listener.onFailure(new IllegalArgumentException("Empty parameters for QueryPlanningTool: " + parameters));
             return;
         }
-        if (!parameters.containsKey(PROMPT_FIELD)) {
-            parameters.put(PROMPT_FIELD, defaultPrompt);
+        if (!parameters.containsKey(SYSTEM_PROMPT_FIELD)) {
```

Comment:
With this PR , we don't allow customer override prompt in QueryPlanningTool, only allow system prompt, right ?